services/service_initializer.go

```diff
@@ -0,0 +1,117 @@
+package services
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/kurtosis-tech/kurtosis-go/kurtosis_service"
+	"github.com/palantir/stacktrace"
+	"net"
+	"os"
+	"path/filepath"
+)
+
+const (
+	ipPlaceholder = "KURTOSISSERVICEIP"
+)
+
+// TODO We MIGHT be able to remove this struct entirely
```

Comment:
is this TODO actionable? should we have this kind of a thing as a TODO?

Reply:
Fixed in accumulated PR comments